lib/utils.js

```diff
@@ -800,6 +800,25 @@ module.exports = {
     });
   },
 
+  getMetaProperty(propertyName, ruleInfo, scopeManager) {
```

Comment:
Either this needs to be able handle any property e.g. `meta.schema` or it should be renamed `getMetaDocsProperty` since it only handles docs properties right now.